FoundationsMastery/CustomString.cs

```diff
@@ -0,0 +1,62 @@
+﻿using FoundationsMastery.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FoundationsMastery
+{
+    public class CustomString : Interfaces.ICustomString
+    {
+        
+
+        public CustomString(char[] mychars)
+        {
+            Contents = mychars;
+        }
+
+        public CustomString()
+        {}
+
+        public IEnumerable<char> Contents
+        {
+            get; set;
+        }
+
+        public int Length
+        {
+            get;
```

Comment:
What is this "getting"? What other property in this class has a `Length` or a `Count` ability?